lib/Storage/S3StorageClient.h

```diff
@@ -49,6 +51,7 @@ class S3StorageClient : public StorageClient {
   getWritableFile(llvm::StringRef Path, ContentEncoding Encoding) override;
 
   llvm::Error commit() override;
+  llvm::Error setCredentials(llvm::StringRef Credentials) override;
```

Comment:
The credentials string is opaque and storage-backend specific.
Document its format here.

Reply:
Documented